src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -1139,6 +1152,8 @@ def get_crash_data(crash_data, symbolize_flag=True):
 
   is_kasan = 'KASAN' in crash_stacktrace_without_inlines
   is_golang = '.go:' in crash_stacktrace_without_inlines
+  is_python = '.py", line' in crash_stacktrace_without_inlines
```

Comment:
Not that "Uncaught Python exception" ?

Reply:
"Uncaught python exception" might not be the only thing that we want to catch, so I aimed at the most generic way to detect python crashes.